### PR TITLE
[Icons] Fix sass deprecation warnings

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Cleanup the new `bwicons -> styles.scss` file that has deprecation warnings revolving around the use of `/` for division.  Use the `calc()` function since the `math.div` function is not available in our current version of `sass`

## Code changes
- **styles.scss:** Surrounded all division math with the `calc()` function. Removed unused `export` call

## Testing requirements
- Make sure icons using either `bwi-fw, bwi-lg, bwi-ul, bwi-li, or bwi-li & bwi-lg` still appear the same throughout the client applications (once the updated `jslib` has been applied)

## Before you submit
- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: angular/src/scss/bwicons/styles/style.scss [hunk 1]
```diff
@@ -84,12 +84,12 @@ $icomoon-font-path: "~@bitwarden/jslib-angular/src/scss/bwicons/fonts/" !default
 
 .bwi-li {
   position: absolute;
-  left: -(30em / 14);
-  width: (30em / 14);
-  top: (2em / 14);
+  left: calc(30em / 14) * -1;
```
Comment (cscharf) [inline angular/src/scss/bwicons/styles/style.scss:87]: Wouldn't the `* -1` go inside of the `calc()` function? Order or operations for deriving a negative result shouldn't matter in these cases anyway.

File: angular/src/scss/bwicons/styles/style.scss [hunk 2]
```diff
@@ -84,12 +84,12 @@ $icomoon-font-path: "~@bitwarden/jslib-angular/src/scss/bwicons/fonts/" !default
...
+  width: calc(30em / 14);
+  top: calc(2em / 14);
   text-align: center;
   &.bwi-lg {
-    left: -(30em / 14) + (4em / 14);
+    left: (calc(30em / 14) * -1) + calc(4em / 14);
```
Comment (cscharf) [inline angular/src/scss/bwicons/styles/style.scss:92]: Same

File: angular/src/scss/bwicons/styles/style.scss (hunk 1 above)
Comment (Hinton) [reply angular/src/scss/bwicons/styles/style.scss:87]: Do we need the `* -1` trick or does calc support that natively using `calc(-30em/14)`?

Comment (vincentsalucci) [reply angular/src/scss/bwicons/styles/style.scss:87]: After doing some digging it appears that as long as we supply a unit of measurement the negative value will be calculated properly. I will update the PR to remove  `* -1` trick

**Review by Hinton — APPROVED** [review]

**Pull request merged**